src/content/configuration/experiments.mdx

```diff
@@ -164,6 +164,14 @@ Define the location to store the lockfile.
 
 By default webpack would generate a `<compiler-name.>webpack.lock` file>. Make sure to commit it into a version control system. During the `production` build, webpack will build those modules beginning with `http(s):` protocol from the lockfile and caches under [`experiments.buildHttp.cacheLocation`](#experimentsbuildhttpcachelocation).
 
+#### experiments.buildHttp.proxy
+
+Define the proxy to specify the proxy server for remote resources.
+
+- Type: `string`
+
+By default webpack would try to use the `http_proxy` environment variable to specify the proxy server for remote resources. Besides, you can define the proxy server by setting the `proxy` option.
```

Comment:
```suggestion
By default webpack would imply the proxy server to use for fetching remote resources from the `http_proxy` environment variable. However, you can also specify one through the `proxy` option.
```

Should it be `https_proxy`?

Reply:
Confirmed that using `http_proxy`.

Reply:
Is this case sensitive? or both `http_proxy`/`HTTP_PROXY` works?

Reply:
Works for me. /cc @chenxsan